src/Tokenizer.ts

```diff
@@ -625,7 +625,8 @@ export class _Tokenizer {
     const nextChar = match[1] || match[2] || '';
 
     if (!nextChar || !prevChar || this.rules.inline.punctuation.exec(prevChar)) {
-      const lLength = match[0].length - 1;
+      // unicode Regex counts emoji as 1 char; spread into array for proper count (used multiple times below)
```

Comment:
Instead of the "used multiple times below" comment, can we refactor this into a function such as `countUnicodeChars(str: string)`?

Reply:
For something that is single-line and only happens 3 times, I figured the speed overhead of wrapping it in a function wasn't worth it.

Reply:
Yeah and I guess the 3rd usage is different. Ok lets ship it <picture data-single-emoji=":shipit:" title=":shipit:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/squirrel/465f40c0e0.png" alt=":shipit:" align="absmiddle"></picture>

Reply:
Performance wise:

- **Creating a function**: so small you won't even notice it
- **Using the spread operator**: bad

Reply:
@MathiasWP do you know a better way to count Unicode characters in JavaScript?